### DISCUSSION TITLE
Can't connect to mobile wallets on prod

### DISCUSSION THREAD

**Discussion by thxvatsal** [WalletConnect/walletconnect-monorepo#2802] [Developer Support] [answered]
Hi!

I am using the latest `@walletconnect/ethereum-provider` and `@walletconnect/modal` for my library [Wallet01](https://github.com/Huddle01/Wallet01/pull/24).

I have written and example app and it works fine locally, although when deployed [here](https://wallet01-git-walletconnectv2-huddle01.vercel.app/) it refuses to connect to metamask or rainbow wallet.

Although, metamask remains unresponsive after scanning, the rainbow wallet returns `*Connection Failed*: Session timed-out`.

**Comment by adrianotadao** [comment, marked as answer]
hey @thxvatsal 

the only think I know is that metamask has a bug https://github.com/rainbow-me/rainbowkit/issues/1262 =l which I am waiting to deploy the migration to production.

**Reply by thxvatsal** [reply]
Thanks @adrianotadao for replying! I guess I'll wait for it 😁

**Reply by finessevanes** [reply]
Hey @adrianotadao @thxvatsal can you all confirm you're on the latest wallet version and SDKs?
